shared/actions/config/index.tsx

```diff
@@ -384,9 +383,13 @@ const routeToInitialScreen = (state: Container.TypedState) => {
 
     // A share
     if (state.config.startupSharePath) {
+      // TODO/FIXME: this seems unused. [ShareDataIntent] in
+      // actions/platform-specific/push.native.tsx happens at cold-start too,
+      // so maybe we can just use that. Though that happens before this
+      // (routeToInitialScreen), so in the end it just routes to the saved tab
+      // which gets rid of the destination-picker modal.
```

Comment:
Any thought on this? Cold-start share still doesn't work and I think this is the reason.

Reply:
Cold start might not work currently, but I have a fix here: https://github.com/keybase/client/pull/19047

If it works on a non-cold start my fix should fix it